### PR TITLE
:sparkles: wfp-787: added case details

### PR DESCRIPTION
**Pull request by katie-cawthorne**

### CODE REVIEW

File: server/views/pages/summary.njk [hunk 1]
```diff
@@ -102,6 +102,59 @@
...
+                        value: {
+                        html: '<span class="moj-badge moj-badge--red moj-badge--large">Missing</span>'
+                    }
+                    },
+                    {
+                        key: {
+                        html: 'Pre-sentence report' + '<br><span class="govuk-!-font-size-16 govuk-!-font-weight-regular">' + data.courtReport.description + '</span>'
+                    },
+                        value: {
+                        text: data.courtReport.completedDate | dateFormat
```
Comment (carlov20) [inline server/views/pages/summary.njk:141]: There is a chance there might not be a court report so we should cater for that scenario by displaying the Missing badge

File: server/views/pages/summary.njk [hunk 2]
```diff
@@ -102,6 +102,59 @@
...
+                    }
+                    },
+                    {
+                        key: {
+                        text: 'Last OASys assessment'
+                    },
+                        value: {
+                        text: data.assessment.lastAssessedOn | dateFormat
```
Comment (carlov20) [inline server/views/pages/summary.njk:149]: I'm unsure if this scenario is only in lower environments but there is a chance the assessment might not exist too

Comment (katie-cawthorne) [reply server/views/pages/summary.njk:149]: Ok, no probs, I'll do that now 👍

File: server/views/pages/summary.njk (hunk 1 above)
Comment (katie-cawthorne) [reply server/views/pages/summary.njk:141]: Ok no probs, I'll add that now 👍

**Review by carlov20 — APPROVED** [review]
👍

**Pull request merged**